examples/go.mod

```diff
@@ -0,0 +1,8 @@
+module example
+
+go 1.12
```

Comment:
go 1.12 ?
kk is 1.13 , so could we update this to 1.13?

Reply:
ideally, yes.